services/cloudaccounts/k8s/k8s.go

```diff
@@ -143,3 +150,15 @@ func (service *Service) EnableImageAssurance(body ImageAssuranceEnableRequest) (
 
 	return resp, nil
 }
+/*
+	flow-logs
+*/
+func (service *Service) EnableFlowLogs(body FlowLogsEnableRequest) (*http.Response, error) {
+	relativeURL := fmt.Sprintf("%s/%s/%s", cloudaccounts.RESTfulPathK8S, cloudaccounts.RESTfulPathK8SFlowLogs, cloudaccounts.RESTfulPathK8sEnable)
```

Comment:
As we discussed offline, this seems to be the old API.
Please consult with Yehoda about what we want to do here in this regard.